PokeAlarm/Alarms/Discord/DiscordAlarm.py

```diff
@@ -233,12 +236,15 @@ def send_alert(self, alert, info):
                     'lat': info['lat'],
                     'lng': info['lng']
                 }
+                static_map_url = replace(alert['map'],
+                                         coords if not
+                                         isinstance(alert['map'], str)
+                                         else info)
+                if self.__signing_secret_key is not None:
```

Comment:
This ensures that gmaps always gets used as long as they have a secret key set in their config, regardless of their alarm settings; it might also force-enable gmaps (not sure here)

Best would be to follow the alarm map setting first, then gmaps if it's not set